xml/System.Diagnostics/Process.xml

```diff
@@ -2989,7 +2989,9 @@ The calling process is a member of the associated process' descendant tree.</exc
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The value returned by this property represents the current size of nonpaged system memory used by the process, in bytes. System memory is the physical memory used by the operating system, and is divided into paged and nonpaged pools. Nonpaged memory allocations remain in system memory and are not paged out to the virtual memory paging file.  
+ The value returned by this property represents the most recently refreshed size of nonpaged system memory used by the process, in bytes. To get the current size, you need to call <xref:System.Diagnostics.Process.Refresh> method first.
```

Comment:
"To get the most up to date size, .." might be better wording. "To get the current size.." reads like, unless I call Refresh(), I might get bogus results or no results at all. (Sort of an oerload of meaning of "current")

Reply:
How about "To get a more recent size"?